source/_components/notify.telegram.markdown

```diff
@@ -84,10 +84,16 @@ notify:
     chat_id: CHAT_ID_2
 ```
 
-Configuration variables:
-
-- **name** (*Optional*): Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
-- **chat_id** (*Required*): The chat ID of your user.
+{% configuration %}
+name:
+  description: Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
```

Comment:
The default value is mentioned in the description.

Reply:
Not sure actually. But if you're going to use ``` `notify` ```, be aware that you'll have to quote it.

Reply:
Have it left out, maybe that @frenck could give more clarity.